groups/alteryx-sandbox/locals.tf

```diff
@@ -18,6 +18,12 @@ locals {
   alteryx_worker_ami_id       = var.alteryx_worker_ami_id == "" ? data.aws_ami.alteryx_worker_ami[0].id : var.alteryx_worker_ami_id
   alteryx_worker_ami_owner_id = local.account_ids_secrets["shared-services"]
   alteryx_subnet_id           = [for o in data.aws_subnet.alteryx : o.id if o.tags["Name"] == "sub-alteryx-a" || o.tags["Name"] == "sub-alteryx-b"]
```

Comment:
While not part of this PR:

As you're now using specific subnets, this can be replaced with a single data lookup for the specific subnet in question. E.g. `data.aws_subnet.alteryx.id`